customizations.md

```diff
@@ -0,0 +1,12 @@
+# Customizations
+Based on your company, there are essential customizations that depend on your preferences and location. 
+
+
+### Timezone
+Set a timezone in your Looker Admin located at the top right corner. Scheduled Looks will be run in the time zone of your choosing. To set a database querying time zone, you’ll have to go into Connections on the sidebar. 
+
+### Local Currency
+In the [sales](sales.view.lkml) view, choose a  value_format_name based on your local currency or currency preference.
+
+### Color Palette
+You can change the color palette by {{ FILL IN BLANK }}
```

Comment:
pick colors from here:

https://polaris.shopify.com/visuals/colors#section-color-palette

choose several that look good together to you.